torch/_dynamo/eval_frame.py

```diff
@@ -389,6 +389,15 @@ def _fn(*args, **kwargs):
                 else:
                     return fn(*args, **kwargs)
 
+            if torch.jit.is_tracing():
+                if config.error_on_nested_jit_trace:
```

Comment:
If we're using a config to make this toggleable, should we mention the config in the runtime error?

Reply:
Idk I just made it symmetric with the other config lol